bundle/compliance/cis_azure/rules/cis_4_1_2/rule.rego

```diff
@@ -0,0 +1,19 @@
+package compliance.cis_azure.rules.cis_4_1_2
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+
+finding = result {
+	# filter
+	data_adapter.is_sql_server
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(is_public_access_disabled),
+		{"Resource": data_adapter.resource},
+	)
+}
+
+is_public_access_disabled {
+	data_adapter.properties.publicNetworkAccess == "Disabled"
+} else = false
```

Comment:
> Ensure that `StartIpAddress` is not set to `0.0.0.0`, `/0` or other combinations which allows access to wider public IP ranges including Windows Azure IP ranges.
    Also ensure that `FirewallRuleName` doesn't contain 
     `AllowAllWindowsAzureIps` which is the rule created when the `Allow Azure services and resources to access this server` setting is enabled for that SQL Server.

I don't see how this check follows the benchmark audit